src/TinyLittleMvvm.Demo/Views/WindowView.xaml.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MahApps.Metro.Controls;
+
+namespace TinyLittleMvvm.Demo.Views
+{
+    /// <summary>
+    /// Interaction logic for WindowView.xaml
+    /// </summary>
+    public partial class WindowView : MetroWindow
+    {
+        public WindowView()
+        {
+            InitializeComponent();
+        }
+    }
+}
```

Comment:
Delete this file, it is not required. (all windows and controls in TinyLittleMvvm don't have a code-behind file)

Reply:
Why is that?